en/flight_controller/crazyflie2.md

```diff
@@ -160,11 +161,26 @@ Then, you need to stick the battery on top of the SD card deck using a double si
 | IMU_ACCEL_CUTOFF  | 30                          |
 | IMU_DTERM_CUTOFF  | 70                          |
 | SYS_FMU_TASK      | Enabled                     |
+| SDLOG_PROFILE     | 1                           |
+| EKF2_ABL_LIM      | 2.0                         |
 
-\* Since we haven't been able to fly it with optical flow, we don't use optical flow data yet.
+## Altitude Control 
 
+Crazyflie is able to fly in *Altitude* mode if you use a [Z-ranger deck](https://store.bitcraze.io/collections/decks/products/z-
+ranger-deck). According to the datasheet, the maximum height (above ground) the range finder can sense is 2 m. However, when tested on dark surfaces this value decreases to 0.5 m. On a light floor, it goes up to max 1.3 m. This means you cannot hold altitudes above this value in *Altitude* or *Position* flight modes.
```

Comment:
This link is broken because it crosses multiple lines. See rendered output.

Reply:
"This means you cannot hold altitudes above this value in *Altitude* or *Position* flight modes.". IMO this should be a warning.